### PR TITLE
Fix invalid expression syntax

### PR DESCRIPTION
**Pull request by fgreinacher**
The expression had an extra '$' character which made it always evaluate to true.
    
See https://github.com/boostsecurityio/poutine/blob/main/docs/content/en/rules/if_always_true.md.
    
CLA: trivial

:hammer_and_pick: with :heart: by [Siemens](https://opensource.siemens.com/)

### CODE REVIEW

**Review by levitte — APPROVED** [review]

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ jobs:
       run: nmake test VERBOSE_FAILURE=yes TESTS=-test_fuzz* HARNESS_JOBS=4
     - name: install
       # Run on 64 bit only as 32 bit is slow enough already
-      if: $${{ matrix.platform.arch == 'win64' }}
+      if: matrix.platform.arch == 'win64'
```
Comment (nhorman) [inline .github/workflows/windows.yml:69]: I see from the docs that the bare format is allowed in if: statements, but wouldn't it be more consistent to do what the rest of the expressions in this file do, and use the evaluation syntax, i.e.:
if: ${{ matrix.platform.arch }} == 'win64'

Comment (fgreinacher) [reply .github/workflows/windows.yml]: @nhorman Adapted!

**Review by nhorman — APPROVED** [review]

**Comment by nhorman** [comment]
thank you!  And concur with cla:trival

**Review by shahsb — APPROVED** [review]

**Review by paulidale — APPROVED** [review]
agreed trivial

**Comment by t8m** [comment]
Merged to the master branch. Thank you for your contribution.

**Pull request closed without merging**